<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>linux on H&amp;W</title>
    <link>http://hyyfrank.github.io/tags/linux/</link>
    <description>Recent content in linux on H&amp;W</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <lastBuildDate>Sat, 07 Jul 2018 11:36:45 +0800</lastBuildDate><atom:link href="http://hyyfrank.github.io/tags/linux/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Crontab Issue Tracking</title>
      <link>http://hyyfrank.github.io/post/backend/play-with-contab/</link>
      <pubDate>Sat, 07 Jul 2018 11:36:45 +0800</pubDate>
      
      <guid>http://hyyfrank.github.io/post/backend/play-with-contab/</guid>
      <description>how to track issue sudo /var/log/syslog | grep cron  capture the output 1 2 * * * /home/hyy/Start.py &amp;gt;/tmp/output.log 2&amp;gt;&amp;amp;1  check cron is running ps -ef | grep cron | grep -v grep  check the path is correct crontab is running in with cwd == $HOME, if you using python os.getcwd() in different folder structure, make sure you cd to the correct place or else it will affect your code by output &amp;ldquo;can not find module&amp;rdquo; stuff.</description>
    </item>
    
  </channel>
</rss>
